### PR TITLE
added pylint to codacy config

### LINKED ISSUE #425: Add configurations in .codacy.yml
## Description
Codacy has to be properly configured for the project to ensure the reviews reflects our standards.

## Acceptance Criteria
- [x] Add python engines to configuration
- [x] Exclude folder not required in reviews

## Definition of Done
- [x] All of the required items are completed.
- [x] Approval by 1 mentor.

## Estimation
1 hour

### PR DESCRIPTION
**Pull request by Fenn-CS**
Signed-off-by: Fenn-25 <fenn25.fn@gmail.com>

### Description
Added configuration for .codacy.yml

Fixes #425

### Type of Change:
- Quality Assurance

### How Has This Been Tested?
Non code change codacy check observed

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials

### CODE REVIEW

File: .codacy.yml [hunk 1]
```diff
@@ -1,2 +1,20 @@
...
+   enabled: true
+
+ coverage:
+   enabled: true
+
+languages:
+   python:
+
 exclude_paths:
+
```
Comment (MehaKaushik) [inline .codacy.yml]: Add a new line at the end of the file.

Comment (nida) [reply .codacy.yml]: @Fenn-CS Is @MehaKaushik s review incorporated?

Comment (Fenn-CS) [reply .codacy.yml]: Ok, will try to follow up with all of the open issues soon. Currently have exams

**Comment by sakshi1499** [comment]
@sammy1997 Do you think this can be merged?

**Comment by nida** [comment]
Let me know if you want me to review

**Comment by sakshi1499** [comment]
Closing this PR because it's outdated.

**Pull request closed without merging**